charter-template.html

```diff
@@ -188,14 +188,24 @@ <h3>
           </p>
           <p class="todo">Consider making a second list for existing specifications that the group will maintain.</p>
           <dl>
-            <dt id="web-foo" class="spec"><a href="#">Web <i class="todo">[spec name]</i></a></dt>
+            <dt id="web-foo" class="spec"><a href="#"><i class="todo">[new spec name]</i></a></dt>
+            <dd>
+              <p>This specification will define <i class="todo">[concrete description]</i>.</p>
+
+              <p class="draft-status"><b>Draft state:</b> <i class="todo">[No draft | <a href="#">Use Cases and Requirements</a> | <a href="#">Editor's Draft</a> | <a href="#">Member Submission</a>]</i></p>
+              <p class="milestone"><b>Expected completion:</b> <i class="todo">[Q1–4 yyyy]</i></p>
+              <p class=todo>If this is a new technical report that will start as a copy of a different TR:</p>
+              <p><b>Initial Text:</b> The <span class="todo">title, stable URL, and publication date of the existing technical report which will serve as the initial text of the deliverable.</span>
+              <p class=todo>Otherwise, if there is an existing <a href='https://www.w3.org/policies/process/#exclusion-draft'>Exclusion Draft</a>, use the [current spec name] template below instead.</p>
```

Comment:
This _might_ be confusing, since there's often an existing Exclusion Draft of the source TR even if the group is making a copy into a new TR. It does say "Otherwise" to maybe head off that confusion, but that raises the question of what to do with a spec that isn't new, but also doesn't yet have an Exclusion Draft. Or I might be overthinking this.